docs/user_guide/assessment.rst

```diff
@@ -408,11 +408,56 @@ to take advantage of the inherent plotting capabilities of
 .. literalinclude:: ../auto_examples/plot_quickstart.py
     :language: python
     :start-after: # Analyze metrics using MetricFrame
+    :end-before: # Customize plots with `ylim`
 
 .. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_001.png
     :target: auto_examples/plot_quickstart.html
     :align: center 
 
+It is possible to customize the plots. Here are some common examples:
+
+Customize Plots: :code:`ylim`
+'''''''''''''''''''''''''''''
+The y-axis range is automatically set, which can be misleading, therefore it is
+sometimes useful to set the `ylim` argument to define the yaxis range.
+
+.. literalinclude:: ../auto_examples/plot_quickstart.py
+    :language: python
+    :start-after: # Customize plots with `ylim`
+    :end-before: # Customize plots with `colormap`
+
+.. figure:: ../auto_examples/images/sphx_glr_plot_quickstart_002.png
+    :align: center
+
+
+Customize Plots: :code:`colormap`
```

Comment:
Sorry, I actually meant to add `:code:` only in the text, not in headers. It seems to be causing some issues when I run it locally.

Reply:
OK I'll take it back, perhaps it's the length of the header underline. But it's definitely struggling with the backquotes...

```
fairlearn\docs\user_guide\assessment.rst:434: WARNING: Title underline too short.

Customize Plots: :code:`colormap`
''''''''''''''''''''''''''''''''
```
I suggest replacing them with `~`

Reply:
sorry for the delay, got a bit held up last week. I reflected the feedback and it seems to be passing the build now. Does it look okay? Thank you!